server.js

```diff
@@ -38,30 +39,33 @@ server.listen(port)
 server.on('error', onError)
 server.on('listening', onListening)
 
+// @ts-ignore
+
 /**
- * Event listener for HTTP server "error" event.
+ * Event listener for HTTP server 'error' event.
  */
 
+// @ts-ignore
 function onError (error) {
   if (error.syscall !== 'listen') {
     throw error
   }
 
-  const bind = typeof port === 'string'
-    ? 'Pipe ' + port
-    : 'Port ' + port
+  const bind = typeof port === 'string' ? 'Pipe ' + port : 'Port ' + port
```

Comment:
could you tell what could be the possible values of port?

Reply:
Hi Akshay, I don't have any idea about this. In my PR this line spacing got changed by prettier so its says I have made those changes but I haven't.

Reply:
Fine 😅.